assets/js/modules/tagmanager/datastore/__factories__/builders.js

```diff
@@ -361,6 +401,26 @@ const analyticsTagAMP = ( propertyID, { accountId, containerId } = {} ) => {
 };
 /* eslint-enable sitekit/acronym-case */
 
+export const buildLiveContainerVersion = ( {
```

Comment:
This is a little off as the others are either `Web` or `AMP` suffixed, and this is still a web container. I see why it's separate but ideally the name would be a bit more self-descriptive, or maybe we could find a way to merge this with the existing utility. Not a blocking issue though.